main/src/main/java/cgeo/geocaching/filters/core/TypeGeocacheFilter.java

```diff
@@ -9,14 +9,13 @@ public class TypeGeocacheFilter extends ValueGroupGeocacheFilter<CacheType, Cach
 
     public TypeGeocacheFilter() {
         //gc.com groups in their search their cache types as follows:
-        //* "Tradi" also includes: GCHQ, PROJECT_APE
-        //* "Event" also includes:CITO,mega,giga, gps_exhibit,commun_celeb, gchq_celeb, block_party
+        //* "Unknown" also includes: Unknown, GCHQ, PROJECT_APE
+        //* "Celebration Event" also includes: mega,giga, gps_exhibit,commun_celeb, gchq_celeb, block_party
         //-> unlike gc.com, currently CITO is an OWN search box below (to make number even)
-        addDisplayValues(CacheType.TRADITIONAL, CacheType.TRADITIONAL, CacheType.GCHQ, CacheType.PROJECT_APE);
-        addDisplayValues(CacheType.EVENT, CacheType.EVENT, CacheType.MEGA_EVENT, CacheType.GIGA_EVENT, CacheType.COMMUN_CELEBRATION,
+        addDisplayValues(CacheType.OTHER, CacheType.UNKNOWN, CacheType.GCHQ, CacheType.PROJECT_APE);
```

Comment:
To prevent the need of creating nonexisting cache types:
* pick ONE of the cache types in the visual "group" (eg for "other", use "unknown"; for "specialevent" use "mega")
* in method "valueToUserDisplayableValue" (this is where visualization is done) return for these "group cache types" the correct text